migrations/2_deploy_contracts.js

```diff
@@ -240,6 +310,44 @@ module.exports = function (deployer, network, accounts) {
                     segmentPayment: deployConfigs.segmentPayment,
                 }
             );
+        } else if (networkName === 'alfajores') {
+            const lendingPool = poolConfigs.lendingPool;
+
+            // Deploys GoodGhostingContract
+            await deployer.link(SafeMathLib, GoodGhostingCeloContract);
+            await deployer.deploy(
+                GoodGhostingCeloContract,
+                inboundCurrencyAddress,
+                lendingPoolAddressProvider,
+                deployConfigs.segmentCount,
+                deployConfigs.segmentLength,
+                segmentPaymentWei,
+                deployConfigs.earlyWithdrawFee,
+                deployConfigs.customFee,
+                lendingPool,
+                deployConfigs.merkelroot
+            );
+
+            // Prints deployment summary
+            printSummaryCelo(
+                {
+                    inboundCurrencyAddress,
+                    lendingPoolAddressProvider,
+                    segmentCount: deployConfigs.segmentCount,
+                    segmentLength: deployConfigs.segmentLength,
+                    segmentPaymentWei,
+                    earlyWithdrawFee: deployConfigs.earlyWithdrawFee,
+                    customFee: deployConfigs.customFee,
+                    lendingPool,
+                    merkelRoot: deployConfigs.merkelroot
+                },
+                {
+                    networkName,
+                    selectedProvider: deployConfigs.selectedProvider,
+                    inboundCurrencySymbol: deployConfigs.inboundCurrencySymbol,
+                    segmentPayment: deployConfigs.segmentPayment,
```

Comment:
isn't this the same as mainnet deployment? Do we really need to duplicate everything or could we just check the network to pass the correct contract only?

Reply:
@fodisi the constructor params are different I mean the number is the same but the contract types aren't same there is no data provider needed in celo but yeah I can try to  make it minimal

Reply:
Ok, thanks for clarifying it @viraj124 . I believe it's OK for now.

Reply:
thanks can you give your review then on the pr